### PR TITLE
SAK-43136 Replaced primary color with standard bg color for Samigo Create page plus additional visual and layout tweaks

### PR DESCRIPTION
**Pull request by fostersdesign**
https://jira.sakaiproject.org/browse/SAK-43136

* Changed background colours of the panels on the Add Assessments page
* Fixed headings, labels, and spacing on the Add Assessments page
* Fixed heading on the Trash page
* Fixed some CSS in tool_sam.css

See SAK-43136 for before and after screenshots.

### CODE REVIEW

File: samigo/samigo-app/src/webapp/css/tool_sam.css [hunk 1]
```diff
@@ -111,14 +111,10 @@ h4.samigo-category-subhead-2 {
   padding:.3em
 }
 
-/*
-.navModeQuestion
-{background-color:#CCC;text-align:left;padding:.3em;vertical-align:middle font-size:1.2ems}
-*/
 .navViewAction {
-background-color:#CCC;
+background-color: #eee;
```
Comment (profmikegreene) [inline samigo/samigo-app/src/webapp/css/tool_sam.css:115]:
```suggestion
background-color: var(--sakai-background-color-4);
```

File: samigo/samigo-app/src/webapp/css/tool_sam.css [hunk 2]
```diff
@@ -129,8 +125,8 @@ font-size:1.2em
 .number {
   width:23px;
   height:23px;
-        FONT-WEIGHT: bold;
-        COLOR:#FFFFFF;
+        font-weight: bold;
+        color:#FFFFFF;
```
Comment (profmikegreene) [inline samigo/samigo-app/src/webapp/css/tool_sam.css:129]:
```suggestion
        color: var(--sakai-text-color-inverted;
```

File: samigo/samigo-app/src/webapp/css/tool_sam.css [hunk 3]
```diff
@@ -416,13 +412,6 @@ ul.ui-tabs-nav li a {
 ul.ui-tabs-nav li.ui-tabs-selected a {
 	color: #000000;
```
Comment (profmikegreene) [inline samigo/samigo-app/src/webapp/css/tool_sam.css:413]:
```suggestion
	color: var(--sakai-text-color-1);
```

File: samigo/samigo-app/src/webapp/css/tool_sam.css [hunk 4]
```diff
@@ -416,13 +412,6 @@ ul.ui-tabs-nav li a {
...
 }
-#authorIndexForm\:import {
-	display: block;
-	margin-top: 12px;
-}
-#authorIndexForm\:createnew {
-	margin-top: 12px;
-}
 tr.list-row-even td, tr.list-row-odd td {
 	border-bottom: 1px solid #C6C6C6;
```
Comment (profmikegreene) [inline samigo/samigo-app/src/webapp/css/tool_sam.css:416]:
```suggestion
	border-bottom: 1px solid var(--sakai-border-color);
```

File: samigo/samigo-app/src/webapp/css/tool_sam.css [hunk 5]
```diff
@@ -507,8 +496,8 @@ a.hideDivision {
 }
 
 .answerkeyFeedbackCommentLabel {
-	FONT-WEIGHT: bold;
-    COLOR:#860000;
+    font-weight: bold;
+    color: #860000;
```
Comment (profmikegreene) [inline samigo/samigo-app/src/webapp/css/tool_sam.css:500]:
```suggestion
    color: var(--warnBanner-color);
```

**Review by profmikegreene — APPROVED** [review]
I'm ok with leaving the CSS colors in for the moment, but if you decide to fix them I've suggested what I'd use.

File: samigo/samigo-app/src/webapp/css/tool_sam.css (hunk 1 above)
Comment (fostersdesign) [reply samigo/samigo-app/src/webapp/css/tool_sam.css]: Good suggestion! Thanks. Looks much better.

File: samigo/samigo-app/src/webapp/css/tool_sam.css (hunk 2 above)
Comment (fostersdesign) [reply samigo/samigo-app/src/webapp/css/tool_sam.css]: Thanks.

File: samigo/samigo-app/src/webapp/css/tool_sam.css (hunk 3 above)
Comment (fostersdesign) [reply samigo/samigo-app/src/webapp/css/tool_sam.css]: Thanks.

File: samigo/samigo-app/src/webapp/css/tool_sam.css (hunk 4 above)
Comment (fostersdesign) [reply samigo/samigo-app/src/webapp/css/tool_sam.css:416]: Thanks.

File: samigo/samigo-app/src/webapp/css/tool_sam.css (hunk 5 above)
Comment (fostersdesign) [reply samigo/samigo-app/src/webapp/css/tool_sam.css:500]: I'm not sure about this one. Since it is out-of-scope of this PR, I'm going to skip it for now.

**Comment by fostersdesign** [comment]
Thanks for the suggestions, @profmikegreene . I've made most of the changes.

File: samigo/samigo-app/src/webapp/css/tool_sam.css [hunk 6]
```diff
@@ -111,14 +111,10 @@ h4.samigo-category-subhead-2 {
...
-/*
-.navModeQuestion
-{background-color:#CCC;text-align:left;padding:.3em;vertical-align:middle font-size:1.2ems}
-*/
 .navViewAction {
-background-color:#CCC;
+background-color: var(--sakai-background-color-4);
 text-align:left;
-padding:.3em;
+padding: 10px;
```
Comment (profmikegreene) [inline samigo/samigo-app/src/webapp/css/tool_sam.css:117]: Should this use 12 or 16 px to be inline with our other spacing ratios?

File: library/src/morpheus-master/sass/modules/tool/samigo/_samigo.scss [hunk 7]
```diff
@@ -16,13 +16,29 @@
...
-		@extend .bs-callout-primary;
-		border-left-width: 1px;
-		margin-top: 20px;
+		margin-bottom: $standard-spacing;
+		padding: $standard-spacing;
+		border: 1px solid var(--sakai-border-color);
+		background: var(--sakai-background-color-2);
+
+		h2 {
+			margin: 0 0 15px 0; 	// to match the .form-group's spacing
```
Comment (profmikegreene) [inline library/src/morpheus-master/sass/modules/tool/samigo/_samigo.scss:25]: 15 not 16?

**Review by profmikegreene — APPROVED** [review]
No blockers for me. Left a couple questions but I think this can go in regardless.

File: samigo/samigo-app/src/webapp/css/tool_sam.css (hunk 6 above)
Comment (fostersdesign) [reply samigo/samigo-app/src/webapp/css/tool_sam.css:117]: Good point. I'll take another look at this.

File: library/src/morpheus-master/sass/modules/tool/samigo/_samigo.scss (hunk 7 above)
Comment (fostersdesign) [reply library/src/morpheus-master/sass/modules/tool/samigo/_samigo.scss:25]: This is from Bootstrap, so it's their number "15".

**Review by ern — APPROVED** [review]

**Pull request merged**